doc/classes/NavigationAgent2D.xml

```diff
@@ -219,7 +219,7 @@
 			Does not affect normal pathfinding. To change an actor's pathfinding radius bake [NavigationPolygon] resources with a different [member NavigationPolygon.agent_radius] property and use different navigation maps for each actor size.
 		</member>
 		<member name="simplify_epsilon" type="float" setter="set_simplify_epsilon" getter="get_simplify_epsilon" default="0.0">
-			The path simplification amount in worlds units.
+			The path simplification amount in world units.
```

Comment:
Reminds me of https://github.com/godotengine/godot/pull/107165 which lacks consensus. Either way it's worth noting that the Navigation classes are the only ones referring to "world units", and even then quite sparingly, which renders this definition somewhat ambiguous.